packages/playground/windows/playground/MainPage.xaml.cpp

```diff
@@ -115,3 +117,14 @@ void MainPage::UpdatePaneCommandState() {
     removePaneCommand->RaiseCanExecuteChanged();
   }
 }
+
+std::unordered_map<std::string, std::string> params;
+
+void MainPage::OnNavigatedTo(Windows::UI::Xaml::Navigation::NavigationEventArgs ^ e) {
```

Comment:
Consider adding a TextBox to the UI (See HostingPane.xaml) to hold this value rathe than plumbing through a command-line arg.  UI would be more discoverable.

Reply:
I thought about this but if we are ever to have a closed inner loop where I can debug from VS we need to be able to pass this in a programmatic, non interactive way

---
In reply to: [339308950](https://github.com/microsoft/react-native-windows/pull/3535#discussion_r339308950) [](ancestors = 339308950)